src/NzbDrone.Core/Configuration/IConfigService.cs

```diff
@@ -94,5 +94,8 @@ public interface IConfigService
 
         CertificateValidationType CertificateValidation { get; }
         string ApplicationUrl { get; }
+
+        // Network
+        bool TrustCGNAT { get; set; }
```

Comment:
For config file/environment variable this should be in `IConfigFileProvider`, the name should follow existing naming, `TrustCgnat`